ui/public/locales/en.json

```diff
@@ -1807,6 +1807,7 @@
 "label.rolename": "Role",
 "label.roles": "Roles",
 "label.roletype": "Role Type",
+"label.rootdisksize": "Root disk size (GB)",
```

Comment:
Also may not be needed, can we use 'label.root.disk.size' instead? Is this one being used?